src/algebra/invertible.lean

```diff
@@ -168,6 +168,9 @@ inv_of_eq_right_inv (by simp)
 (is_unit_of_invertible (2:α)).mul_right_inj.1 $
   by rw [mul_sub, mul_inv_of_self, mul_one, bit0, add_sub_cancel]
 
+@[simp] lemma add_halves'' [semiring α] [invertible (2 : α)] : (⅟2 : α) + (⅟2 : α) = 1 :=
```

Comment:
I wonder if the name `add_halves_of_invertible` might be better here?

Reply:
`inv_of_two_add_inv_of_two` seems like the obvious choice, which is consistent with [`one_sub_inv_of_two`](https://leanprover-community.github.io/mathlib_docs/algebra/invertible.html#one_sub_inv_of_two)

Reply:
The existing `add_halves` lemmas are all about `x / 2` not `1 / 2`

Reply:
I renamed the two lemmas to `inv_of_two_add_inv_of_two` and `inv_of_two_smul_add_inv_of_two_smul`, I agree it's more descriptive.